pkgs/applications/networking/mailreaders/thunderbird/default.nix

```diff
@@ -60,25 +60,29 @@
 , waylandSupport ? true
 , libxkbcommon, calendarSupport ? true
 
-, # If you want the resulting program to call itself "Thunderbird" instead
-# of "Earlybird" or whatever, enable this option.  However, those
-# binaries may not be distributed without permission from the
-# Mozilla Foundation, see
-# http://www.mozilla.org/foundation/trademarks/.
-enableOfficialBranding ? false
+, # As stated by Sylvestre Ledru (@sylvestre) on Nov 22, 2017 at
```

Comment:
```suggestion
# As stated by Sylvestre Ledru (@sylvestre) on Nov 22, 2017 at
```

Reply:
These are not addressed in your PR. Did you forgot to force push?

Reply:
I've force pushed now.